support-frontend/assets/components/footerCompliant/BackToTop.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import {
+  brandBackground,
+  brandText,
+  brandAlt,
+} from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+
+const iconHeight = '42px';
+
+const iconContainer = css`
+    position: relative;
+    float: right;
```

Comment:
Is float actually needed here? I think the position absolute below will handle all this for you.

Reply:
Float enables the white circle behind the icon to actually have an intrinsic width and height, as it's an inline element; using `display: inline-block` works the same but requires more faffing about to position it relative to the text, whereas with float it just sits where it's supposed to.

Reply:
Okay cool.